tutorials/early_stopping/early_stopping.ipynb

```diff
@@ -0,0 +1,1067 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "a5c5a7d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "val_acc = MyTensorboardMetric(\n",
+    "    name=\"val_acc\",\n",
+    "    curve_name=\"val_acc\",\n",
+    "    lower_is_better=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4f3ba5d",
+   "metadata": {},
+   "source": [
+    "## Setting up the OptimizationConfig\n",
+    "\n",
+    "The `OptimizationConfig` specifies the objective for Ax to optimize."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "ada66cf3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "opt_config = OptimizationConfig(\n",
+    "    objective=Objective(\n",
+    "        metric=val_acc,\n",
+    "        minimize=False,\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57aa9cf7",
+   "metadata": {},
+   "source": [
+    "## Defining an Early Stopping Strategy\n",
+    "\n",
+    "A `PercentileEarlyStoppingStrategy` is a simple method that stops a trial if its performance falls below a certain percentile of other trials at the same step (e.g., when `percentile_threshold` is 50, at a given point in time, if a trial ranks in the bottom 50% of trials, it is stopped). \n",
+    "- We make use of `normalize_progressions` which normalizes the progression column (e.g. timestamp, epochs, training data used) to be in [0, 1]. This is useful because one doesn't need to know the maximum progression values of the curve (which might be, e.g., the total number of data points in the training dataset).\n",
+    "- The `min_progression` parameter specifies that trials should only be considered for stopping if the latest progression value is greater than this threshold.\n",
+    "- The `min_curves` parameter specifies the minimum number of completed curves before early stopping will be considered. This should be larger than zero if `normalize_progression` is used. In general, we want a few completed curves to have a baseline for comparison."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "949e8ab5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "percentile_early_stopping_strategy = PercentileEarlyStoppingStrategy(\n",
+    "    # stop if in bottom 50% of runs at the same progression\n",
+    "    percentile_threshold=50,\n",
+    "    # the trial must have passed `min_progression` steps before early stopping is initiated\n",
+    "    # note that we are using `normalize_progressions`, so this is on a scale of [0, 1]\n",
+    "    min_progression=0.3,\n",
+    "    # there must be `min_curves` completed trials and `min_curves` trials reporting data in\n",
+    "    # order for early stopping to be applicable\n",
+    "    min_curves=3,\n",
+    "    # specify, e.g., [0, 1] if the first two trials should never be stopped\n",
+    "    trial_indices_to_ignore=None,\n",
```

Comment:
I don' think we need to talk about this if we don't use it in the tutorial - less is more; folks will always find this in the docstring
```suggestion
```